app/src/views/OperationalLearnings/Filters/index.tsx

```diff
@@ -42,6 +47,15 @@ function Filters() {
         filter: {},
     });
     const handleExportClick = useCallback(() => setFilter({}), [setFilter]);
+    const showActions = Object.values(rawFilter).some((value) => value !== undefined);
+
+    const selectedFilters = Object.entries(rawFilter)
+        .filter(([value]) => value !== undefined)
+        .map(([label, value]) => (
+            <div key={label}>
+                { value}
+            </div>
+        ));
```

Comment:
We need to filter the list of regions, countries, sectors, and disaster types using the filtered values. In this way, we can get the labels for each value.

Reply:
Instead of generating JSX this way, we could simply extract the labels and map over them within the return statement.